README.md

```diff
@@ -1 +1,25 @@
-# status-network
\ No newline at end of file
+# status-network
+
+### Testing locally
+
+Make sure you have node.js installed first.
+
+1. Open Terminal and navigate to the project root directory,
+
+2. Run `yarn install`,
+
+3. We share common elements across our sites using submodules, so you'll need to get those too:
+
+    ```bash
+    git submodule update --init --recursive
+    cd themes/navy/layout/partial/shared-partials
+    git checkout master
+    git pull
+    cd ../../../../../
+    ```
+
+4. Run `./node_modules/.bin/gulp build`
```

Comment:
We should make this into a `package.json` script, and upgrade to Gulp 4. I guess I can do it in a separate PR.

Reply:
Thanks! I used this old gulp command to build the website